packages/design-system/src/tokens/ods/size.yaml

```diff
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resize-step:
+      value: 12rem
+    default:
```

Comment:
The default is not in use. Delete it?

Reply:
Good point, thx. It's meant to be used for non-resizable tiles (like the ones on the spaces-projects view), I added the token without actually using it in the component 🙈 see latest commit in this PR

Reply:
aaaah, now it makes sense 😅